io.jamulus.Jamulus.metainfo.xml

```diff
@@ -21,6 +21,7 @@
     <content_attribute id="social-audio">intense</content_attribute>
   </content_rating>
   <releases>
+    <release version="_3_7_0beta1" date="2021-02-18"/>
```

Comment:
@barthalion previous version (that I added manually) includes an "r" at the beginning. Can I change anything to make the auto update include that, or should I change the previous one to remove the "r"?

Reply:
what i'm wondering is why it's picking the pre-releases as well

Reply:
It is marking the correctly as pre-releases in https://release-monitoring.org/project/155434/, so I guess it wouldn't be a big problem. We can have those and merge them into the `beta` branch once we have that

Reply:
re: `r` prefix, as long as CI does not complain, I think it's up to you.